src/sentry/notifications/utils/participants.py

```diff
@@ -171,6 +171,16 @@ def get_owners(project: Project, event: Event | None = None) -> Iterable[Team |
         outcome = "match"
         recipients = ActorTuple.resolve_many(owners)
 
+    ownership = ProjectOwnership.objects.filter(project=project).first()
+    # Used to supress extra notifications to all project members, only notify the would-be auto-assignee
+    if (
+        recipients
+        and ownership
+        and not ownership.fallthrough
+        and not features.has("organizations:notification-all-recipients", project.organization)
+    ):
+        return [recipients[-1]]
```

Comment:
re: the slice: thats cool, thankss

re: if `recipients` is a `set`: It has 3 lines declaring it above and only one resolves with an empty set, which is won't make it to this line since the first check of the conditional above is `recipients`. The other two return lists i believe
 
```python
    if not owners:
        outcome = "empty"
        recipients = set()

    elif owners == ProjectOwnership.Everyone:
        outcome = "everyone"
        recipients = User.objects.filter(id__in=project.member_set.values_list("user", flat=True))

    else:
        outcome = "match"
        recipients = ActorTuple.resolve_many(owners)
```